src/interface/obsidian/src/utils.ts

```diff
@@ -134,12 +179,20 @@ export async function updateContentIndex(vault: Vault, setting: KhojSetting, las
     if (error_message) {
         new Notice(error_message);
     } else {
+        if (userTriggered) new Notice('✅ Updated Khoj index.');
```

Comment:
Ah, seems like this would better capture the different indexing pathways.

Reply:
Yeah, putting it here allows only showing the indexing success notice if not error and user triggered indexing manually (e.g via Khoj obsidian plugin settings page)